experiments/AMIP/moist_mpi_earth/coupler_driver.jl

```diff
@@ -93,8 +100,23 @@ import ClimaCoupler
 pkg_dir = pkgdir(ClimaCoupler)
 COUPLER_OUTPUT_DIR = joinpath(pkg_dir, "experiments/AMIP/moist_mpi_earth/output", joinpath(mode_name, run_name))
 !isdir(COUPLER_OUTPUT_DIR) && mkpath(COUPLER_OUTPUT_DIR)
-REGRID_DIR = joinpath(COUPLER_OUTPUT_DIR, "regrid_tmp/")
+
+# create unique REGRID_DIR for each buildkite job
+if "BUILDKITE" in keys(ENV)
+    if ClimaComms.iamroot(comms_ctx)
+        REGRID_DIR = joinpath(COUPLER_OUTPUT_DIR, "regrid_tmp/", ENV["BUILDKITE_STEP_KEY"])
+    else
```

Comment:
do you need this `else`? if it isnt the root process, is REGRID_DIR = "" used/should it be used?

not sure I follow this part.

Reply:
If it isn't the root process, it will receive the `REGRID_DIR` defined on the root process. I added this `else` to set `REGRID_DIR=""` just so the variable is initialized in both branches, but the empty string will be overwritten by the broadcast on line 113.